resource/README.md

```diff
@@ -0,0 +1,41 @@
+# Binary Resources
+
+This directory contains binary resources needed by cockroach in various
+capacities, includes images and documentation for display on our github page.
+
+## Embedded UI
+
+The /ui directory contains the client-side code for cockroach's web admin
+console. These files are embedded into the cockroach binary via the
+[go-bindata](https://github.com/jteeuwen/go-bindata) package, which is used to
+generate the `embedded.go` file in this directory.
+
+#### Modification
+If any modifications are made to the contents of the /ui directory, run `make`
+in this directory to generate a new `embedded.go`, which should _then be
+committed along with the original files_. The generated file is committed
+because making our web admin resource requires [additional non-go
+dependencies](####dependencies)
+
+#### Development
+While actively developing the user interface, use `make debug` in order to run
+go-bindata in debug mode.  This will instruct go-bindata to proxy your local
+files (rather than using embedded versions) so you'll be able to edit them live
+without recompiling or restarting the server.  
+
+Note that only those files that were present the last time you ran `make debug`
+will be proxied; if you add a new file, you must run recompile and restart the
+server after running `make debug`.
+
+Before committing, be sure to run `make` to generate a non-debug version of
+`embedded.go`. This is enforced by our build system, but forgetting to do this
```

Comment:
I think the final tests build the deploy image without the files on disk, so the tests would fail then, but I wouldn't say it's strictly "enforced", especially as we rebuild CircleCI that might well get lost.